schema.sql

```diff
@@ -1,5 +1,47 @@
 /* Database schema to keep the structure of entire database. */
 
+/* FEATURE BRANCH 03-querying-multiple-tables */
+
+-- A) Create new table called OWNERS with columns id, full_name & age.
+CREATE TABLE owners
+(
+    id serial NOT NULL,
+    full_name character varying(120) NOT NULL,
+    age integer,
+    PRIMARY KEY(id)
+);
+
+-- B) Create new table called SPECIES with columns id & name.
+CREATE TABLE species
+(
+    id serial NOT NULL,
+    name character varying(100) NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- C) MODIFY animals table, make sure the id column is an autoincremented
+--    PRIMARY KEY; remove column species; add column species_id (foreign key);
+--     add column owner_id (foreign key).
+ALTER TABLE animals
+DROP COLUMN species,
+ADD owwner_id integer,
```

Comment:
- [ ] Please make sure that your `schema` query is typo-free. It seems like your column name should be `owner_id` not **owwner_id**. Because of this small typo, there is a significant change in the table structure and an error caused by it.